internal/bot.go

```diff
@@ -0,0 +1,70 @@
+package internal
+
+import (
+	auth "github.com/Floor-Gang/authclient"
+	util "github.com/Floor-Gang/utilpkg"
+	dg "github.com/bwmarrin/discordgo"
+	"log"
+)
+
+// Bot structure
+type Bot struct {
+	Auth   *auth.AuthClient
+	Client *dg.Session
+	Config *Config
+}
+
+// Start starts discord client, configuration and database
+func Start() {
+	var err error
+
+	// get Config.yml
+	config := GetConfig()
+
+	// setup authentication server
+	// you can use this to get the bot's access token
+	// and authenticate each user using a command.
+	authClient, err := auth.GetClient(config.Auth)
+
+	if err != nil {
+		log.Fatalln("Failed to connect to authentication server", err)
+	}
+
+	register, err := authClient.Register(
+		auth.Feature{
+			Name:        "Stop Redirect", // Give this bot / feature a name
+			Description: "Looks for links to check if they are redirecting to another web site.", // Describe what this bot is doing
+			Commands: []auth.SubCommand{ // list all the commands this bot / feature has
+				{
+					Name:        "help",           // Command name like "add"
+					Description: "This command describes what the bot does.",           // Describe what the command does
+					Example:     []string{""}, // [command name, argument 1, argument 2] like [add, #channel, #channel]
+				},
```

Comment:
Remove default comments like "Give this bot / feature a name"

Reply:
done